### DISCUSSION TITLE
Exception messages should be corrected.

### DISCUSSION THREAD

**Discussion by theyasub** [CommunityToolkit/Maui#614] [General]
When I reviewed the project, I noticed many small exception syntax errors in some places. Letter case should be default case, old version is pascal case.
What do you think, can I fix this little mistake?

**Comment by GeorgeLeithead** [comment]
Do you have examples of where this occurs and what you thing the correct case should be for the examples?

**Reply by VladislavAntonyuk** [reply]
It is a part of #616

**Comment by CliffAgius** [comment]
Closed as answered
